documentation/release_notes.markdown

```diff
@@ -12,6 +12,20 @@ canonical: "/puppetserver/latest/release_notes.html"
 
 For release notes on versions of Puppet Server prior to Puppet Server 2.5, see [docs.puppet.com](https://docs.puppet.com/puppetserver/2.4/release_notes.html).
 
+## Puppet Server 2.8.0
+
+Released August 31, 2017
+
+This is a feature release of Puppet Server.
+
+> **Warning:** If you're upgrading from Puppet Server 2.4 or earlier and have modified `bootstrap.cfg`, `/etc/sysconfig/puppetserver`, or `/etc/default/puppetserver`, see the [Puppet Server 2.5 release notes first](#potential-breaking-issues-when-upgrading-with-a-modified-bootstrapcfg) **before upgrading** for instructions on avoiding potential failures.
```

Comment:
What are the settings in sysconfig/default that this effects?

Reply:
[JVM memory allocation](https://docs.puppet.com/puppetserver/latest/install_from_packages.html#memory-allocation) and [max heap size](https://docs.puppet.com/puppetserver/latest/tuning_guide.html#jvm-heap-size).